how_to_build.md

```diff
@@ -16,9 +16,7 @@ Ensure that the following are installed:
 
 1. .NET 3.5, required by ILMerge
 
-1. .NET framework 4.6.1 or later

```

Comment:
I think this is implied by VS2017 (you can't have VS2017 without having net461 or later)